SODA.Utilities.Tests/ExcelDataReaderHelperTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Data;
+using System.IO;
+using System.Linq;
+using Microsoft.Win32;
+using NUnit.Framework;
+using SODA.Utilities.Tests.Mocks;
+using Excel;
+
+namespace SODA.Utilities.Tests
+{
+    [TestFixture]
+    public class ExcelDataReaderHelperTests
+    {
+        [Test]
+        [Category("ExcelDataReaderHelper")]
+        public void MakeConnection_With_Empty_Filename_Throws_Exception()
```

Comment:
Please update all of the test names to match the public method under test (e.g. `GetRowsFromDataSheets`)